lib/exercism/submission.rb

```diff
@@ -103,6 +103,7 @@ def like!(user)
   def unlike!(user)
     likes.where(user_id: user.id).destroy_all
     self.is_liked = liked_by.length > 0
+    notifications.unread.where(action: 'like', actor_id: user.id).destroy_all
```

Comment:
Not worries! Just curious hah. I was just super surprised. Thanks for the review. Should have those changes and tests soon.